### PR TITLE
ADD webserver.ps1 script for Windows

### PR DESCRIPTION
**Pull request by codewizard13**

### CODE REVIEW

File: webserver.ps1 [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Install Chocolatey
+Write-Host "Installing Chocolatey ..." -BackgroundColor DarkMagenta
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+
+# Install VirtualBox with multipass (including extenstions and guest additions)
+Write-Host "Installing Multipass with VirtualBox as hypervisor ..."  -BackgroundColor DarkMagenta
+choco install virtualbox multipass
+choco install virtualbox.extensionpack
```
Comment (stephenlauck) [inline webserver.ps1]: could line 8 and 9 just be rolled into line 7 something like:

`choco install virtualbox multipass virtualbox.extensionpack virtualbox-guest-additions-guest.install`

File: webserver.ps1 [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+choco install virtualbox-guest-additions-guest.install
+
+# Run multipass to create vm (launch)
+Write-Host "Create the Virtual Machine ..."  -BackgroundColor DarkMagenta
+muLtipass launch --network Wi-Fi --cpus 1 --disk 20G --mem 2G # --name slick-machine
+multipass set local.driver=virtualbox
```
Comment (stephenlauck) [inline webserver.ps1]: should we set the local setting BEFORE launching the instance?

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: webserver.ps1 (hunk 1 above)
Comment (codewizard13) [reply webserver.ps1]: Made required changes

File: webserver.ps1 (hunk 2 above)
Comment (codewizard13) [reply webserver.ps1]: Made required changes

Comment (codewizard13) [reply webserver.ps1]: Working on further changes requested by @stephenlauck:

- Wrap line 7 in IF statement (if virtualbox already installed)
- See if you can choco install version of virtualbox
- How to upgrade existing VMs to work with new version

**Review by stephenlauck — APPROVED** [review]

**Pull request merged**